wiki/overrides/partials/tabs-item.html

```diff
@@ -48,7 +48,7 @@
             </div> {{ title }}
         </a>
         <div class="dropdown-container">
-            <div class="dropdown-content">
+            <div class="glass-backdrop">
```

Comment:
Can you revert this change? I changd quite a lot in dropdown-content to get the blur to work on both.
dropdown-content uses its own blur css